webpack/assets/javascripts/react_app/redux/reducers/users/personalAccessTokens/index.js

```diff
@@ -0,0 +1,51 @@
+import {
+  USERS_PERSONAL_ACCESS_TOKEN_FORM_BUTTON,
+  USERS_PERSONAL_ACCESS_TOKEN_FORM_SUCCESS,
+  USERS_PERSONAL_ACCESS_TOKEN_FORM_OPENED,
+  USERS_PERSONAL_ACCESS_GET_REQUEST,
+  USERS_PERSONAL_ACCESS_GET_SUCCESS,
+  USERS_PERSONAL_ACCESS_GET_FAILURE
+
+} from '../../../consts';
+import Immutable from 'seamless-immutable';
+
+const initialState = Immutable({
+  isOpen: false,
+  isSuccessful: false,
+  body: '',
```

Comment:
I'm having a hard time with this variable name - can we call it something else? what does body means?